bin/cyhy-import

```diff
@@ -79,6 +80,21 @@ def has_intersections(db, nets, filename, owner):
         print_intersections(intersections)
         return True
 
+def has_restricted_ips(nets):
+    reader = geoip2.database.Reader('/usr/local/share/GeoIP/GeoIP2-City.mmdb')
+    bad_countries = ['China','Russia','Iran','North Korea']
+    has_bad_address = False
```

Comment:
I don't like the use of "good" and "bad" when involving real world entities as they're words that represent value judgements. I suggested restricted as a replacement as it aligns with the method name you chose.

```suggestion
    restricted_countries = ['China','Russia','Iran','North Korea']
    has_restricted_address = False
```

Reply:
Good suggestion!

Reply:
Yes, I like that. I am bad at coming up with variable names haha